src/Common/Twig/TwigExtension.php

```diff
@@ -282,7 +282,12 @@ function ($string) {
             new TwigFilter(
                 'dateToTime',
                 function ($str) {
-                    return strtotime($str);
+                    // handle vestigial dates of 0000-00-00 00:00:00
+                    if ((strtotime($str)) <= 0) {
+                        return strtotime('now');
```

Comment:
will this still be needed with your vitals upgrade database fix? (especially considering your upgrade database fix will give an accurate date/time where this will just give the current date/time)
(If we expect this to happen in other places, then prob best to fix where it's happening like did in vitals.)

Reply:
yeah, was wondering about that, okay, will revert this